components/sup/src/manager/commands.rs

```diff
@@ -298,7 +297,7 @@ pub fn service_load(
                         req,
                         &source,
                         &service_spec.bldr_url,
-                        &service_spec.channel,
+                        &ChannelIdent::from(&service_spec.channel),
```

Comment:
For consistency, `ServiceSpec` should return a `ChannelIdent`, rather than a String.

Reply:
Changed the `channel` type in `ServiceSpec` in https://github.com/habitat-sh/habitat/pull/6074/commits/cbebb204f2ee3b0b2cafe139cbf99ed1baf73d94#diff-67161b6446fbddc099ba24f430c21d4bR283, allowing this [to be simplified](https://github.com/habitat-sh/habitat/pull/6074/commits/0a5cdd959beebcade60238a09d13bf21ea9a5c2e) to `&service_spec.channel`